easybuild/easyconfigs/s/Short-Pair/Short-Pair-20170125-Python3fix.patch

```diff
@@ -0,0 +1,204 @@
+Makes Short-Pair compatible with Phyton3
+Author: Filip Kružík (INUITS)
```

Comment:
This patch file was created through an updated `Short-Pair.py` that comes from someone else, you should mention that clearly here (and mention where the updates script was taken from, and a which commit ID).

Rather than mentioning yourself as patch "author", it would be more accurate to say "patched created by" imho